pom.xml

```diff
@@ -104,6 +104,7 @@
     <maven.license.plugin.version>2.3.0</maven.license.plugin.version>
     <!-- Library Dependency Versions -->
     <accumulo.version>1.10.4</accumulo.version>
+    <tomcat.version>9.0.82</tomcat.version>
```

Comment:
can we use a newer version, works with [10.0.27](https://mvnrepository.com/artifact/org.apache.tomcat/tomcat-jasper/10.0.27) for example